controller/space_test.go

```diff
@@ -402,6 +403,25 @@ func (rest *TestSpaceREST) TestListSpacesOK() {
 	}
 }
 
+func (rest *TestSpaceREST) TestListSpacesOKEmptyResult() {
+	// delete all existing spaces
+	svc, ctrl := rest.SecuredController(testsupport.TestIdentity)
+	_, existingSpaces := test.ListSpaceOK(rest.T(), svc.Context, svc, ctrl, nil, nil, nil, nil)
+	spaceRepository := space.NewRepository(rest.DB)
+	for _, existingSpace := range existingSpaces.Data {
+		rest.T().Log("Deleting existing space with ID=", existingSpace.ID.String())
+		spaceRepository.Delete(svc.Context, *existingSpace.ID)
```

Comment:
So, you are deleting all spaces. Even the default one. I guess it will cause problems.